src/components/v5/common/Pills/MotionStateBadge/consts.ts

```diff
@@ -1,6 +1,9 @@
 import { MotionState } from '~utils/colonyMotions.ts';
 
-export const MOTION_STATE_TO_CLASSNAME_MAP: Record<MotionState, string> = {
+export const MOTION_STATE_TO_CLASSNAME_MAP: Record<
+  MotionState | 'Unknown',
```

Comment:
If `Unknown` is already included in the `MotionState` enum per the changes in `src/utils/colonyMotions.ts`, then this `| 'Unknown'` shouldn't be needed here and it can be safely removed.